include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -101,9 +204,9 @@ class fpga_policy : public device_policy<KernelName>
 // Starting with c++17 we can simply define sycl as inline variable.
 #    if _ONEDPL___cplusplus >= 201703L
 
-inline device_policy<> dpcpp_default{};
+inline device_policy<> dpcpp_default{__internal::__global_instance_tag{}};
```

Comment:
If `_ONEDPL_PREDEFINED_POLICIES` is not set, then the `__global_instance_tag` struct is not declared/defined. Would that be a problem for this line here and the fpga one below?

Reply:
The macro is checked at the line 97 above, just not shown in the diff by default.